### PR TITLE
LWIMIQ post 2017-10-02

### PR DESCRIPTION
**Pull request by skateman**
@JPrause @cybette

### CODE REVIEW

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello everyone, my name is [Dávid Halász](https://github.com/skateman) and I'd like to guide you through the changes we made in the last week.
+
+## Featured
+
+During the weekend [Roman](https://github.com/romanblanco), [Julian](https://github.com/juliancheal) and myself were attending at [EuRuKo 2017](https://www.euruko2017.org/) in Budapest. It is a beautiful city where you can among others eat [halászlé](https://hu.wikipedia.org/wiki/Hal%C3%A1szl%C3%A9) (not named after me) and see the [Halászbástya](https://hu.wikipedia.org/wiki/Hal%C3%A1szb%C3%A1stya) (also not named after me).
```
Comment (cybette) [inline site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: wondering if it's better to use the English wikipedia links? Up to you...

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Improved
+
+While [Ladas](https://github.com/ladas) improved how we [save metrics](https://github.com/ManageIQ/manageiq/pull/15976) and [Tim](https://github.com/imtayadeway) added [event streams](https://github.com/ManageIQ/manageiq/pull/16021) as a new product feature we were exploring the Square of Heroes.
+
+![The Square of Heroes](https://i.imgur.com/UjJGoB9.jpg)
+
+## Fixed
+
+Back in the day, the *Service UI* was called *Self Service* and we missed some places were it should've been renamed. Thanks to [Allen](https://github.com/AllenBW) now it is [fixed and consistent](https://github.com/ManageIQ/manageiq-ui-service/pull/1030) across the repository. As we moved our GTL code to Angular, the policy simulation screen got broken and did not display quadicons but thanks to [Milan](https://github.com/mzazrivec) this is now [fixed](https://github.com/ManageIQ/manageiq-ui-classic/pull/2034). In the meantime we explored the square around the Matthias Church.
```
Comment (cybette) [inline site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: Would be nice to have a link to quadicons - http://manageiq.org/docs/guides/ui/quadicons (for the benefit of those like me who didn't realise what it means :stuck_out_tongue: )

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md (hunk 1 above)
Comment (skateman) [reply site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: @cybette they're literally translated so won't make that much sense :disappointed:

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+![The Matthias Church](https://i.imgur.com/ATEQCPl.jpg)
+
+## New
+
+While we listened to a talk about failures and why it's bad that ruby is not slow enough, [Chris Hale](https://github.com/chalettu) added the ability to [disable custom buttons](https://github.com/ManageIQ/manageiq-ui-service/pull/1012) in the SUI and Moti added support for [publishing a VM](https://github.com/ManageIQ/manageiq/pull/15981) by oVirt.
```
Comment (cybette) [inline site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: Maybe just "Chris" to be consistent with referring to others by first name only. Also what about github link for Moti?

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md (hunk 1 above)
Comment (skateman) [reply site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: oh my bad, it's there, changing it

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+![Scalability talk](https://i.imgur.com/PUmU7I7.jpg)
+
+## Wrapping up
+
+The changes above are just a few from the many that happened around the project, feel free to check out our [GitHub organization](https://github.com/manageiq) page for more details.
+
+![Buda Castle](https://i.imgur.com/9nHHGji.jpg)
+
+Each year EuRuKo is organized in a different European city that is being chosen by the attendees of the previous conference. We are very happy that the next year it will be in Vienna that is just from an hour ride from our office in Brno. Here I would like to say thank you for the organizers and the speakers, it was an amazing event!
```
Comment (cybette) [inline site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: Minor stuff: "just from an hour ride" should be "just an hour's ride" (or "just a one-hour ride") ; "thank you for" should be "thank you to"

**Comment by cybette** [comment]
Great post, and cool pics! Thank you @skateman, I enjoyed it very much :smile:

**Comment by cybette** [comment]
Thanks for the quick changes @skateman! One more thing, I assume all photos are yours, and if so, you might want to add a license, e.g.:

_All images above by [Dávid Halász](https://github.com/skateman) licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)_

File: site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+published: true
+tags: LWIMIQ
+---
+
+Hello everyone, my name is [Dávid Halász](https://github.com/skateman) and I'd like to guide you through the changes we made in the last week.
+
+## Featured
+
+During the weekend [Roman](https://github.com/romanblanco), [Julian](https://github.com/juliancheal) and myself were attending at [EuRuKo 2017](https://www.euruko2017.org/) in Budapest. It is a beautiful city where you can among others eat [halászlé](https://en.wikipedia.org/wiki/Fisherman%27s_soup) (not named after me) and see the [Halászbástya](https://en.wikipedia.org/wiki/Fisherman%27s_Bastion) (also not named after me).
+
```
Comment (syncrou) [inline site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md]: 😆 !!!

**Pull request merged**